packages/react-components/react-tabster/src/hooks/useTabster.ts

```diff
@@ -14,6 +18,9 @@ export const useTabster = (): TabsterTypes.TabsterCore | null => {
   const { targetDocument } = useFluent();
 
   const defaultView = targetDocument?.defaultView || undefined;
+
+  const shadowDOMAPI = (defaultView as WindowWithTabsterShadowDOMAPI | undefined)?.__tabsterShadowDOMAPI;
```

Comment:
instead of reading a undocumented window global - let's just make `getParent` handle shadow DOM by default. It's a fairly small change, and generally in Fluent we're trying to increase support for shadow dom (e.g. using composedPath)

Reply:
I will adjust it. But we still need to read undocumented global. This is the way to enable ShadowDOM keeping it treeshakeable.

Reply:
Changed to single getParent().